config/channel/consolidated/deployments/controller.yaml

```diff
@@ -50,6 +50,8 @@ spec:
           value: config-leader-election
         - name: DISPATCHER_IMAGE
           value: ko://knative.dev/eventing-kafka/cmd/channel/consolidated/dispatcher
+        - name: SERVICE_ACCOUNT
```

Comment:
how about `DISPATCHER_SERVICE_ACCOUNT` to match the `DISPATCHER_IMAGE` and be more indicating about its actual purpose?

Reply:
I thought about that, but I want to make sure distributed ch and this one uses the same env var keys.
I don't want to touch distributed channel's env vars right now.

Reply:
gotcha, how about then adding a comment to the yaml above that env var explaining what it is?

Reply:
+1